src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java

```diff
@@ -578,16 +578,17 @@ public void generateAlgodIndexerObjects (JsonNode root) throws IOException {
                 for (Map.Entry<String, JsonNode> cls : getSortedSchema(schemas).entrySet()) {
                     JsonNode tags = cls.getValue().get("tags");
                     if (tags != null) {
-                        boolean isPrivate = false;
+                        boolean ignore = false;
                         Iterator<JsonNode> tagIter = tags.elements();
                         while (tagIter.hasNext()) {
                             JsonNode tag = tagIter.next();
-                            if (tag.asText().equals("private")) {
-                                isPrivate = true;
+                            String tagText = tag.asText();
+                            if (tagText.equals("private") || tagText.equals("experimental")) {
```

Comment:
nit: maybe a "shouldIgnore(tagText)" helper to avoid the magic strings in 3 places?

Reply:
Good idea, adopted in c365f1176d800b595f31673f8b33a97b07b67575